_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -53692,6 +53692,11 @@
 /mob/living/simple_animal/sloth/paperwork,
 /turf/open/floor/glass,
 /area/station/cargo/storage)
+"shd" = (
+/obj/structure/lattice,
+/obj/structure/grille,
+/turf/open/space/openspace,
+/area/space/nearstation)
```

Comment:
is it both grille and lattice on other maps?

Reply:
I'm pretty sure yeah, if you go to the lattice above meta sec I think it's both grilles and lattices. I'll double check tomorrow though.

Reply:
found it, yeah, it has a grille spawned and lattice instead of just grilles, but I think its functionally the same minus random damage

![image](https://github.com/tgstation/tgstation/assets/36141254/c2b9a8c5-6330-4765-b172-dfd22ea6076b)

Reply:
okay just making sure since i have dementia and forgor